### PR TITLE
Openshift route is added 🎉

### PR DESCRIPTION
**Pull request by ckavili**
#### Summary
Add Openshift route template and update values file accordingly. 
Fix `imageTag` variable for MySQL version which was misnamed from the subchart.

### CODE REVIEW

File: charts/mattermost-team-edition/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Mattermost Team Edition server.
 name: mattermost-team-edition
-version: 3.9.1
+version: 3.9.2
```
Comment (cpanato) [inline charts/mattermost-team-edition/Chart.yaml:4]: this is a "new feature" need to bump to 3.10.0
```suggestion
version: 3.10.0
```

File: charts/mattermost-team-edition/templates/route.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  port:
+    targetPort: {{ include "mattermost-team-edition.name" . }}
+  tls:
+    termination: edge
+  to:
+    kind: Service
+    name: {{ include "mattermost-team-edition.name" . }}
+    weight: 100
+  wildcardPolicy: None
+  {{- end }}
```
Comment (cpanato) [inline charts/mattermost-team-edition/templates/route.yaml:22]: nit: new line

File: charts/mattermost-team-edition/templates/route.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{{- if .Values.route.enabled -}}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/timeout: 4m
+    openshift.io/host.generated: "true"
+  labels:
+    app: {{ include "mattermost-team-edition.name" . }}
```
Comment (cpanato) [inline charts/mattermost-team-edition/templates/route.yaml:9]: the labels need to be similar as we have here: https://github.com/mattermost/mattermost-helm/blob/master/charts/mattermost-team-edition/templates/deployment.yaml#L6

**Comment by ckavili** [comment]
Hi @cpanato Thanks for the review! Just updated the files based on your comments.

**Pull request merged**